test/Microsoft.AspNet.Hosting.Tests/WebHostBuilderTests.cs

```diff
@@ -40,60 +41,65 @@ public void Build_honors_UseStartup_with_string()
         [Fact]
         public async Task StartupMissing_Fallback()
         {
+            var application = new RequestDelegate(context => { return Task.FromResult(0); });
```

Comment:
This will go away once `UseServer` takes an `IServer`. This hack was added so the `RequestDelegate` passed into the `IServer` could be accessed for use in `AssertResponseContains`.